Makefile

```diff
@@ -39,6 +39,13 @@ build: $(BIN_DIR)/goreleaser
 	$(BIN_DIR)/goreleaser build --snapshot --rm-dist
 .PHONY: build
 
+release-local: $(BIN_DIR)/goreleaser
+	$(BIN_DIR)/goreleaser release --snapshot --rm-dist
+.PHONY: release-local
+
+$(BIN_DIR)/goreleaser:
+	@GOBIN=$(BIN_DIR) go install github.com/goreleaser/goreleaser@latest
```

Comment:
Note that there is a `GORELEASER_VERSION` variable at the top of the Makefile, however this appeared to be unused, and did not relate to a particular version that was accessible via a `go install` command.

Reply:
Just noted a distinct similarity between this change and https://github.com/Versent/saml2aws/pull/798/files.  Somewhere in the middle lies an accurate fix.  It should not be a `.PHONY`, but the version constraint seems to work.  Will update.

Reply:
@duckfez Just a heads up that I hit the same issue as you, and fixed it prior to realising that there was a similar PR up already.